request_test.go

```diff
@@ -2231,6 +2232,418 @@ func TestRequest_UsageChecks(t *testing.T) {
 		req.Expect()
 		req.chain.assertFailed(t)
 	})
+
+	t.Run("Expect after an Expect", func(t *testing.T) {
```

Comment:
Since there are a lot of new subtests, let's create a new test TestRequest_OrderChecks.